### DISCUSSION TITLE
Cobalt Panicked

### DISCUSSION THREAD

**Discussion by ClaudeVonRiegan222** [Raytwo/Cobalt#38] [Issues]
![Screenshot 2023-11-06 195515](https://github.com/Raytwo/Cobalt/assets/150095828/8bde2f39-e61d-408f-bff6-823561a99f37)

Cobalt crashes every time I load up Fire Emblem Engage.
I have no mods.

**Comment by Raytwo** [comment]
Hello, was a bug, please update to 0.36.1
